scripts/install_server_from_src.sh

```diff
@@ -15,67 +14,133 @@ set -e
 : ${DOWNLOAD_CLIENT_TEMPLATES:=true}
 : ${PYTHON_INSTALL:=true}
 : ${DOCKER:=false}
+: ${INSTALL_BIN:="install"}
+: ${INSTALL_OPTS:="-p -m644"}
+: ${SRC_DIR:="/grr"}
+
+function header()
+{
+  echo ""
+  echo "##########################################################################################"
+  echo "     ${*}";
+  echo "##########################################################################################"
+}
+
+OPTIND=1
+while getopts "h?ls:" opt; do
```

Comment:
we still have a weird mix of ways to pass options in now though. We should just do env vars or flags, not both. Suggest you switch yours to env vars.